external_review/external_review_collate.py

```diff
@@ -37,14 +75,19 @@ def collate(geography_level, category):
             if final_df.empty:
                 final_df = ind
             else:
-                final_df = final_df.merge(ind, right_index=True, left_index=True)
+                final_df = final_df.merge(
+                    ind, right_index=True, left_index=True, how="left"
+                )
         except Exception as e:
             print(
                 f"Error merging indicator {ind_name} at geography level {geography_level}"
             )
             raise e
     final_df.index.rename(geography_level, inplace=True)
-    final_df.to_csv(f"external_review/{category}/{category}_{geography_level}.csv")
+    folder_path = f".staging/{category}"
+    if not path.exists(folder_path):
+        makedirs(folder_path)
+    final_df.to_csv(f".staging/{category}/{category}_by_{geography_level}.csv")
```

Comment:
Don't think we need the "_by" here as per @AmandaDoyle

Reply:
Ah ok I can change that. It won't effect the output as the names of the files are changed when copied to DO but good to be consistent